docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/subscribe-aws.rst

```diff
@@ -56,7 +55,10 @@ configuration on the CrateDB Cloud end.
 .. image:: ../../../_assets/img/aws-subscribe-confirm.png
    :alt: CrateDB Cloud on AWS subscription confirmation
 
-Click on the *Set Up Your Account* button to proceed.
+After clicking the *Set Up Your Account*, you will be redirected to the CrateDB
+Cloud console. From here, you can follow the :ref:`usual deployment procedure
+<cluster-deployment-stripe>` use your new AWS subscription for the billing in
+the CrateDB Cloud console.
```

Comment:
Perhaps worth adding a note that AWS Subscriptions can take up to 10 minutes to be confirmed, before they are usable in CrateDB Cloud.

Reply:
Oh, didn't notice that, mine was ready in a couple of seconds. Will add. 
Is that the case also for Azure? @SStorm

Reply:
No, Azure is usually a few seconds, AWS can take a bit longer.